### DISCUSSION TITLE
virus total scanner - youtube music.exe file

### DISCUSSION THREAD

**Discussion by ankitrawat30** [pear-devs/pear-desktop#2289] [Q&A]
i tried downloading youtubemusic.exe file and scanning from virus total it shows bkavpro W32.AIDetectMalware it it false report or true, is it safe to download it or not ?
![Screenshot 2024-07-30 103246](https://github.com/user-attachments/assets/5d0c8d02-e6c4-47d9-98f8-031c64b107fa)

**Comment by ArjixWasTaken** [comment]
Found this thread

https://answers.microsoft.com/es-es/windows/forum/all/falso-positivo-en-virustotal-w32aldetectmalware1/74485413-c752-4935-a6cf-6e35f68cf9d2

Google translate it into English
